### ISSUE TITLE
Create separate icon files

### ISSUE THREAD

**Issue by smeijer** [Templarian/MaterialDesign-JS#4]
Is it an idea to create separate icon files, so that we can just import the icons that we need? https://github.com/Templarian/MaterialDesign-JS/blob/8ecadec56d85820a8912bd9801aba85c5acc2e81/mdi.js is currently 1.44MB in size. I feel bad about including the whole file in my bundle, just to use a few icons.

```diff
import Icon from '@mdi/react';
- import { mdiAccount } from '@mdi/js';
+ import AccountPath from '@mdi/js/Account';
```

**Comment by Templarian** [comment]
A production build in any application will tree shake unused icons.

- http://dev.materialdesignicons.com/getting-started/rollupjs
- http://dev.materialdesignicons.com/getting-started/webpack

**Comment by smeijer** [comment]
I agree that in a perfect world that should happen. However, for example `meteorjs` doesn't do this.

**Comment by Templarian** [comment]
@smeijer I don't know meteorjs, but in 2019 I _highly_ doubt there is any framework that doesn't support tree shaking out of the box. Might just need to enable it. 😕

Can you please double check this. If that really is the case we can look into something, but hopefully it's as easy as us documenting something.

**Comment by smeijer** [comment]
First version was released in 2012, and Meteor was way ahead of it's time. Unfortunately, that's not longer the case.

There is an open ticket to add tree shaking, dating from 22 Oct 2017.

https://github.com/meteor/meteor-feature-requests/issues/206#issuecomment-341117517

_edit_, that being said. In the mean while I discovered https://github.com/levrik/mdi-react, which wraps these icons in React components. Which makes my life even easier.

So don't spend to much time on thinking about this topic.
